### ISSUE TITLE
Crash while clicking 'View Surface' on a planet marked as having ruins 

### ISSUE THREAD

**Issue by Hetshu** [EarendelDevelopers/factorio-mods#253]
After clicking 'View surface' on a ruin planet, I got the follwing  error
```
The mod Space Exploration (0.6.89) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event space-exploration::on_gui_click (ID 1)
__space-exploration__/scripts/ruins/arty40.lua:24: attempt to index field 'radar' (a nil value)
stack traceback:
	__space-exploration__/scripts/ruins/arty40.lua:24: in function 'prebuild'
	__space-exploration__/scripts/ruin.lua:58: in function 'build'
	__space-exploration__/scripts/ruin.lua:301: in function 'zone_build_ruins'
	__space-exploration__/scripts/zone.lua:714: in function 'create_surface'
	__space-exploration__/scripts/zone.lua:996: in function 'get_make_surface'
	__space-exploration__/scripts/remote-view.lua:654: in function 'start'
	__space-exploration__/scripts/zonelist.lua:2546: in function 'callback'
	__space-exploration__/scripts/event.lua:15: in function <__space-exploration__/scripts/event.lua:13>
```

**Comment by Wiwiweb** [comment]
Sorry, this will be fixed next release.

**Comment by InappropriatePenguin** [comment]
Fixed in 0.6.90.
